### PR TITLE
Temporarily substitute wrap and unwrap operations with inserts and moves

### LINKED ISSUE #1563: Temporarily substitute wrap and unwrap operations with inserts and moves
For OT simplifaction purposes, we decided to temporarily use `MoveOperation`s and `InsertOperation`s instead of `WrapOperation` and `UnwrapOperation`. The latter ones are used only by BlockQuote plugin as of now, so this won't have big impact on user experience. We will use moves and inserts until we fix some OT cases with wrap and unwrap.

### PR DESCRIPTION
**Pull request by scofalik**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Removed usage of `WrapOperation` and `UnwrapOperation`, using `MoveOperation` and `InsertOperation` instead. Changed `SplitOperation#insertionPosition` to a settable. Renamed `SplitOperation#position` to `SplitOperation#splitPosition`. Closes #1563.

### CODE REVIEW

**Comment by scofalik** [comment]
https://github.com/ckeditor/ckeditor5-block-quote/pull/25
https://github.com/ckeditor/ckeditor5-undo/pull/93

**Comment by scofalik** [comment]
> Coverage decreased (-0.04%) to 99.96% when pulling b393177 on t/1563 into 0821d90 on master.

100% on my local machine 🤔

**Comment by scofalik** [comment]
These errors are because of block-quote's post fixer is not available in that build.

File: src/model/differ.js [hunk 1]
```diff
@@ -238,9 +232,7 @@ export default class Differ {
 
 				// If the wrap took the element from the graveyard, mark that the element from the graveyard was removed.
```
Comment (pjasiun) [inline src/model/differ.js]: Comment need to be updated.

File: src/model/operation/mergeoperation.js [hunk 2]
```diff
@@ -119,7 +119,12 @@ export default class MergeOperation extends Operation {
 	 * @returns {module:engine/model/operation/splitoperation~SplitOperation}
 	 */
 	getReversed() {
-		return new SplitOperation( this.targetPosition, this.howMany, this.graveyardPosition, this.baseVersion + 1 );
+		const targetPosition = this.targetPosition._getTransformedByMergeOperation( this );
```
Comment (pjasiun) [inline src/model/operation/mergeoperation.js]: Some comment is needed.

File: src/model/operation/splitoperation.js [hunk 3]
```diff
@@ -26,25 +26,27 @@ export default class SplitOperation extends Operation {
 	/**
 	 * Creates a split operation.
 	 *
-	 * @param {module:engine/model/position~Position} position Position at which an element should be split.
+	 * @param {module:engine/model/position~Position} splitPosition Position at which an element should be split.
 	 * @param {Number} howMany Total offset size of elements that are in the split element after `position`.
+	 * @param {module:engine/model/position~Position} insertionPosition Position at which the clone of split element
+	 * (or element from graveyard) will be inserted.
```
Comment (pjasiun) [inline src/model/operation/splitoperation.js]: Maybe some example?

File: src/model/writer.js [hunk 4]
```diff
@@ -653,7 +651,8 @@ export default class Writer {
 		do {
 			const version = splitElement.root.document ? splitElement.root.document.version : null;
 			const howMany = splitElement.maxOffset - position.offset;
-			const split = new SplitOperation( position, howMany, null, version );
+			const insertionPosition = SplitOperation.getInsertionPosition( position );
+			const split = new SplitOperation( position, howMany, insertionPosition, null, version );
```
Comment (pjasiun) [inline src/model/writer.js]: The way you create this operation is too complicated. :/

File: src/model/differ.js (hunk 1 above)
Comment (scofalik) [reply src/model/differ.js]: Mmmm... I don't understand why?

**Comment by pjasiun** [comment]
We should remove the code and tests related to wrap operations the model (position, differ, etc.).

File: src/model/differ.js (hunk 1 above)
Comment (pjasiun) [reply src/model/differ.js]: As I mentioned belove, we should remove all the code related to wrap and unwrap operations which is not used anymore.

**Pull request merged**